dagmc_stats.py

```diff
@@ -48,6 +44,186 @@ def get_entity_ranges(my_core, meshset, entity_types):
     entity_ranges = {}
     for entity_type in entity_types:
         entity_ranges[entity_type] = my_core.get_entities_by_type(meshset, entity_type) 
-
+        if entity_type == 11:
+            entity_ranges['Volumes'] = my_core.get_entities_by_type_and_tag(meshset, types.MBENTITYSET, dagmc_tags['geom_dim'], [3])
+            entity_ranges['Surfaces'] = my_core.get_entities_by_type_and_tag(meshset, types.MBENTITYSET, dagmc_tags['geom_dim'], [2])
+            entity_ranges['Curves'] = my_core.get_entities_by_type_and_tag(meshset, types.MBENTITYSET, dagmc_tags['geom_dim'], [1])
     return entity_ranges
 
+def get_surfaces_per_volume(my_core, entity_ranges):
+    """
+    Get the number of surfaces that each volume in a given file contains
+    
+    inputs
+    ------
+    my_core : a MOAB core instance
+    entity_ranges : a dictionary of the ranges of each tag in a file
+    
+    outputs
+    -------
+    pdf : a histogram that displays the number of surfaces each volume has (matplotlib)
+    """
+    freqs = []
+    for volumeset in entity_ranges['Volumes']:
+        freqs.append(my_core.get_child_meshsets(volumeset).size())
+    stats = {}
+    stats['minimum'] = min(freqs)
+    stats['maximum'] = max(freqs)
+    stats['median'] = np.median(freqs)
+    stats['average'] = np.average(freqs)
+    return stats
+
+def make_verts_dictionary(vert_list):
+    """
+    returns a dictionary of frequency of the values of the triangles per vertex (ie {number of triangles per vertex: occurences})
+    inputs
+    -----
+    vert_list : a list of all vertices in a file
+    
+    outputs:
+    --------
+    sorted_dict: a dictionary that has information about triangles per vertex, sorted by its keys.
+    """
+    vertices_and_frequencies = {} 
+    #makes dictionary with each vertex and the number of triangles on it
+    for vert in vert_list:
+        number_of_vertex = vert
```

Comment:
Why not just use `vert` here? As the `number_of_vertex` is not used in other place.